### DISCUSSION TITLE
frigate.yml validation errors

### DISCUSSION THREAD

**Discussion by HolgerM1** [blakeblackshear/frigate#4403] [Ask A Question] [answered]
Hi!
Being completely new to frigate I'm stuck for over a day in configuring the frigate.yml. I don't know where the problem is, but with the following frigate.yml:

![Screenshot 2022-11-15 182052](https://user-images.githubusercontent.com/114694650/201987037-d2486d0c-04ae-4c24-939e-b1162c9c2c43.png)

I get the error log: 
![Screenshot 2022-11-15 182018](https://user-images.githubusercontent.com/114694650/201986344-b50d7c0f-a56f-4cfa-a269-8b86c75b6cc3.png)

Could somebody help me out please...

**Comment by NickM-27** [comment, marked as answer]
Looks like you are following a very out of date guide, `clips` is not even a valid role any more. Recommend just using our docs: https://docs.frigate.video/guides/getting_started

**Comment by HolgerM1** [comment]
Thanks on that one. I did not expect it's so complicated to setup frigate. That worked. Maybe you can also help on the next upcoming issue, sorry.
Now I've seen, my coral usb is not recognized?! II'm running RasPI 3b with an coral usb.
![Screenshot 2022-11-15 195521](https://user-images.githubusercontent.com/114694650/202002808-927cf3e2-d06d-4540-bceb-c55d34f037ba.png)

**Reply by NickM-27** [reply]
Are you running home assistant OS

**Reply by HolgerM1** [reply]
yes

**Reply by NickM-27** [reply]
Then please see https://github.com/blakeblackshear/frigate/issues/3890

**Reply by HolgerM1** [reply]
Thanks, will have a look!

**Reply by HolgerM1** [reply]
Great, thank you so much Nick! Protected mode solved it.

**Comment by HolgerM1** [comment]
I'm really sorry, but may I bother you with one more problems. Something seems to be weird with my h264 stream from my Instar 9008FHD cam, I could not figure out the problem. Strange is that the error message only is for the "garage" cam, as the config is identical for both as far as I can see?!
![Screenshot 2022-11-16 130653](https://user-images.githubusercontent.com/114694650/202178710-6e9c0a34-8e22-496d-bdec-0edb61039c87.png)
![Screenshot_20221116_130823](https://user-images.githubusercontent.com/114694650/202178734-1c3ca707-11d3-4ea4-8a01-1e9e358038ef.png)

Edit: Found out that it works when I set the stream to rtsp://admin:PW@IP:554/**12** and change detect to 640x352?! This is the resolution VLC tells me on that stream. But on the other, "11", the resolution is 1920x1080 as I set (according to vlc)?!

**Reply by NickM-27** [reply]
The connection to the camera timed out, if it is a wifi camera might have some interference or other issues. If it doesn't happen often might not be a problem to worry about.

**Comment by HolgerM1** [comment]
Ok, I was thinking there is some wrong configuration, cause of this line:
Could not find codec parameters for stream...

**Reply by NickM-27** [reply]
No, that's ffmpeg saying it couldn't get the type of data in the stream which is normal for when the camera has a spotty connection (since not all the data makes it over WiFi potentially)

**Comment by HolgerM1** [comment]
Cam is cable, I tried it again and firstly it worked. Main problem is, that homeassistant is crashing all the time on the Raspi 3b. Don't know why. I think I try to get it working as docker inside debian on Khadas Vim2.
